### DISCUSSION TITLE
What does the `detekt` gradle task do? (`NO-SOURCE`)

### DISCUSSION THREAD

**Discussion by KotlinIsland** [detekt/detekt#6662] [Q&A] [answered]
I have a Kotlin multiplatform project, and when I run the `detekt` or `check` Gradle job, it just says "NO-SOURCE"

```
> ./gradlew check

...

> Task :compileKotlinJvm UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :jsBrowserProductionExecutableDistributeResources UP-TO-DATE
> Task :jvmTestProcessResources NO-SOURCE
> Task :detekt NO-SOURCE
> Task :processTestResources SKIPPED
> Task :compileKotlinJs
> Task :jsMainClasses

...
```

**Comment by BraisGabin** [comment]
Por means that the talk doesn't have any source to review. So the talk doesn't even need to be run. I assume that your project do have some kotlin files so it should be a problem in the configuration

**Comment by KotlinIsland** [comment, marked as answer]
It's a defect: https://github.com/detekt/detekt/issues/6669
